src/Tizen.NUI.Components/Theme/DefaultThemeCommon.cs

```diff
@@ -449,13 +449,13 @@ public Theme Create()
                 },
                 ActionView = new ViewStyle()
                 {
-                    Size = new Size(-1, 120),
+                    Size = new Size(48, 120),
                     CornerRadius = 0,
                     BackgroundColor = new Color(0, 0, 0, 0),
                 },
                 ActionButton = new ButtonStyle()
                 {
-                    Size = new Size(-1, 120),
+                    Size = new Size(-2, 120),
```

Comment:
Does `-2` mean `Wrap contents`?

Reply:
yes, right.
In ViewStyle there is no Width/HeightSpecification. So Size -1, -2 are used.

Reply:
Thank you to check. I should add them to the ViewStyle.